R/utils.R

```diff
@@ -139,7 +139,7 @@ crosstalk_key <- function() ".crossTalkKey"
 # arrange data if the vars exist, don't throw error if they don't
 arrange_safe <- function(data, vars) {
   vars <- vars[vars %in% names(data)]
-  if (length(vars)) dplyr::arrange_(data, .dots = vars) else data
+  if (length(vars)) dplyr::arrange(data, !!!rlang::syms(vars)) else data
```

Comment:
Does `rlang` automatically import `!!!`? Does it need to be imported/accessed manually?

Reply:
I did end up importing, but I don't think it's necessary:

```r
> rlang::`!!`
function (x) 
{
    abort("`!!` can only be used within a quasiquoted argument")
}
```